src/sonic-host-services/scripts/hostcfgd

```diff
@@ -897,145 +910,179 @@ class HostConfigDaemon:
+
+    def register_callbacks(self):
+        self.subscribe('KDUMP', lambda table, key, op, data: self.kdump_handler(key, op, data), PRIO_INIT)
+        # Handle FEATURE updates before other tables
+        self.subscribe('FEATURE', lambda table, key, op, data: self.feature_handler.handle(key, op, data), PRIO_INIT-1)
+        # Handle AAA, TACACS and RADIUS related tables
+        self.subscribe('AAA', lambda table, key, op, data: self.aaa_handler(key, op, data), PRIO_INIT-2)
+        self.subscribe('TACPLUS', lambda table, key, op, data: self.tacacs_global_handler(key, op, data), PRIO_INIT-2)
+        self.subscribe('TACPLUS_SERVER', lambda table, key, op, data: self.tacacs_server_handler(key, op, data), PRIO_INIT-2)
+        self.subscribe('RADIUS', lambda table, key, op, data: self.radius_global_handler(key, op, data), PRIO_INIT-2)
+        self.subscribe('RADIUS_SERVER', lambda table, key, op, data: self.radius_server_handler(key, op, data), PRIO_INIT-2)
+        # Handle IPTables configuration
+        self.subscribe('LOOPBACK_INTERFACE', lambda table, key, op, data: self.lpbk_handler(key, op, data), PRIO_INIT-3)
+        # Handle NTP & NTP_SERVER updates
+        self.subscribe('NTP', lambda table, key, op, data: self.ntp_global_handler(key, op, data), PRIO_INIT-4)
+        self.subscribe('NTP_SERVER', lambda table, key, op, data: self.ntp_server_handler(key, op, data), PRIO_INIT-4)
+        # Handle updates to src intf changes in radius
+        self.subscribe('MGMT_INTERFACE', lambda table, key, op, data: self.mgmt_intf_handler(key, op, data), PRIO_INIT-5)
+        self.subscribe('VLAN_INTERFACE', lambda table, key, op, data: self.vlan_intf_handler(key, op, data), PRIO_INIT-5)
+        self.subscribe('VLAN_SUB_INTERFACE', lambda table, key, op, data: self.vlan_sub_intf_handler(key, op, data), PRIO_INIT-5)
+        self.subscribe('PORTCHANNEL_INTERFACE', lambda table, key, op, data: self.portchannel_intf_handler(key, op, data), PRIO_INIT-5)
+        self.subscribe('INTERFACE', lambda table, key, op, data: self.phy_intf_handler(key, op, data), PRIO_INIT-5)
+        
         syslog.syslog(syslog.LOG_INFO,
                       "Waiting for systemctl to finish initialization")
         self.wait_till_system_init_done()
         syslog.syslog(syslog.LOG_INFO,
                       "systemctl has finished initialization -- proceeding ...")
 
-        # Update all feature states once upon starting
-        self.feature_handler.update_all_features_config()
-
-        # Defer load until subscribe
-        self.load()
-
-        self.config_db.listen()
+    def start(self):
+        init_load_flag = True
+        while True:
+            state, selectable_ = self.selector.select(SELECT_TIMEOUT)
+            if state == self.selector.TIMEOUT:
+                continue
+            elif state == self.selector.ERROR:
+                raise Exception("Received error from select")
```

Comment:
Could be a one-time error, maybe it is better to proceed rather then fail completelly.

Reply:
Okay, will update